Code/Framework/AzFramework/Platform/Common/Xcb/AzFramework/XcbNativeWindow.cpp

```diff
@@ -219,17 +229,93 @@ namespace AzFramework
         }
     }
 
+    void XcbNativeWindow::DrawSplash()
+    {
+
+        int width = 0;
+        int height = 0;
+
+        auto get_image_data_static = [](int &width, int &height) -> uint8_t*
+        {
+            width = O3DE_LOGO::width;
+            height = O3DE_LOGO::height;
```

Comment:
Both are narrowing implicit casts that raise warnings.
```suggestion
            width = static_cast<int>(O3DE_LOGO::width);
            height = static_cast<int>(O3DE_LOGO::height);
```
Or you can change width and height (and m_width and m_height) to type best matching what is used in xcb calls.